### DISCUSSION TITLE
Price calculation for bundled products with free text input

### DISCUSSION THREAD

**Discussion by hp197** [pretix/pretix#4701] [Ideas] [answered]
Hi,

I would love to see a product with free price input in combination bundled products and calculated prices for those bundled products, based on a calculation scheme.

Use case:
We have an event where we want to offer free price input for a ticket, so people who don't have that much to spend can allow themself to buy a ticket without going through the hassle of mailing us. At the same time, we use bundled products for the different tax rate items we have: a congress ticket at 21% vat, a camping ticket at 9% vat and tourist tax at 0% vat. The congress and camping ticket is each a part of the price and the tourist tax is a fixed part.

Suggestion:
Have a option to set a calculated and/or fixed rate on the bundled products.
In out case that would work out as:
```
Congress ticket: ([$entered price] - €6)/3*2
Camping ticket:  ([$entered price] - €6)/3
Tourist tax:     [fixed] €6
```

This way we can fully comply with tax rules and at the same time have a matching and correct invoice.

**Comment by raphaelm** [comment, marked as answer]
I understand the wish, but it's unlikely to be implemented. The logic around bundled products and their prices/taxes is already so complex and fragile that almost no-one dares to touch it and I don't really want to worsen the situation actively.
